Content.Server/Silicons/Laws/SiliconLawSystem.cs

```diff
@@ -146,6 +157,9 @@ protected override void OnGotEmagged(EntityUid uid, EmagSiliconLawComponent comp
         base.OnGotEmagged(uid, component, ref args);
         NotifyLawsChanged(uid);
         EnsureEmaggedRole(uid, component);
+
+        if(!HasComp<EmaggedComponent>(uid) && component.StunTime != 0)
```

Comment:
you don't need this condition

Reply:
Can borgs not be emagged more than once? I was worried we'd end up with like, perma stunning borgs or something and figured it was safer to keep it

Reply:
it already checks if EmaggedComponent exists before raising the emag event